Weather/Weather/ViewModel/WeatherViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  WeatherViewModel.swift
+//  Weather
+//
+//  Created by Jonathan Zheng on 3/19/24.
+//
+
+import Foundation
+import SwiftUI
+
+class WeatherViewModel: ObservableObject {
+    @Published var weatherData: WeatherModel?
+    @Published var iconImage: UIImage?
```

Comment:
this doesn't need to exist, it's apart of the weatherData

Reply:
WeatherData needs to exist doesnt it?